### PR TITLE
[#17220] - New 64 avatars size

### LINKED ISSUE #17220: Add new 64 sizes for avatars
# Feature Issue

### Description

The QR code component has been fixed [in figma](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=17928-209739&mode=design&t=uBOjm2jYssWHVSQw-4), now the avatars inside them are pointing to the corresponding components. it means new variants (of size 64) have been added.

These are the new designs:

[User avatar](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=28332-1636&mode=design&t=uBOjm2jYssWHVSQw-4)

[Account avatar](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=28174-1966&mode=design&t=uBOjm2jYssWHVSQw-4)

[Channel avatar](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=28351-1662&mode=design&t=uBOjm2jYssWHVSQw-4)

[Wallet avatar](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=114-7665&mode=design&t=uBOjm2jYssWHVSQw-4)

We need to implement them and use them inside the share > QR code component (it just requires to change the given size, which is 32 atm).

### Acceptance Criteria

- The new sizes look as designs
- The QR code component is updated to use the new sizes
- Update their respective preview screens to properly show the new size added.

### PR DESCRIPTION
**Pull request by ulisesmac**
fixes #17220

### Summary

This PR updates the avatars since a new variant  of size 64 have been added. The new variants have no use in the app, so just basic QA is required.

#### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to Quo2 preview screens -> avatar, and test the following in their 64 size:
  - user-avatar
  - wallet-user-avatar
  - channel-avatar
  - account-avatar

status: ready

### CODE REVIEW

File: src/quo2/components/avatars/channel_avatar/style.cljs [hunk 1]
```diff
@@ -5,14 +5,24 @@
 
 (defn outer-container
   [{:keys [size color]}]
-  (let [size (if (= size :size/l) 32 24)]
-    {:width            size
-     :height           size
-     :border-radius    size
+  (let [container-size (case size
+                         :size/l-64 64
```
Comment (J-Son89) [inline src/quo2/components/avatars/channel_avatar/style.cljs]: let's use `:size-64` so 👍 
see discussion and result here
https://github.com/status-im/status-mobile/pull/17279

could you also adjust `:size/l` -> :size-32

Comment (ulisesmac) [reply src/quo2/components/avatars/channel_avatar/style.cljs]: Oh yes, I read  the discussion after having this PR ready. I will fix it across these components 👍

Thank you!

**Comment by ulisesmac** [comment]
@J-Son89 now I'm using the `:size-64` keyword for the new sizes :+1:

File: src/quo2/components/avatars/account_avatar/style.cljs [hunk 2]
```diff
@@ -6,16 +6,20 @@
...
   (case size
-    80 16
-    48 12
-    32 10
-    28 8
-    24 8
-    20 6
-    16 4
+    80       16
+    :size-64 16
```
Comment (J-Son89) [inline src/quo2/components/avatars/account_avatar/style.cljs:16]: @clauxx - sizes is another part of the quo2 refactors that will need to be adjusted. We decided to start using the keyword `:size-<number>` however these uses have to be adjusted for the previous sizes which also means updating this across the uses in the codebase.

Comment (clauxx) [reply src/quo2/components/avatars/account_avatar/style.cljs:16]: 👍 will add an issue which will track adjusting the size across quo2.

**Review by clauxx — APPROVED** [review]
lgtm. Will add a follow-up issue which should take care of adjusting all avatar sizes to this format `size-xx`.

**Review by yqrashawn — APPROVED** [review]

**Comment by churik** [comment]
e2e failures are known due to message reliability and not related to PR.

**Review by Francesca-G — APPROVED** [review]
Looks good ✨

**Pull request merged**